challenges/chat_app_css_illustration/index.html

```diff
@@ -30,33 +30,199 @@
+              >
+                <p class="px-8">30 minute walk</p>
+                <p class="absolute right-6 font-bold">$29</p>
+              </div>
+              <div
+                class="from-light-magenta to-light-violet relative mt-2 flex w-8/12 rounded-lg rounded-bl-sm bg-gradient-to-r py-1 text-white"
+              >
+                <p class="px-8">1 hour walk</p>
+                <p class="absolute right-6 font-bold">$49</p>
+              </div>
+            </div>
+            <div class="text-grayish-blue relative p-2">
+              <input
+                type="text"
+                placeholder="Type a message…"
+                class="w-full rounded-2xl p-2"
+              />
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke-width="1.5"
+                stroke="currentColor"
+                class="bg-desaturated-dark-violet absolute right-6 bottom-4 h-6 w-6 rounded-full"
+              >
+                <path
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  d="M8.25 4.5l7.5 7.5-7.5 7.5"
+                />
+              </svg>
+            </div>
+          </div>
+        </div>
+        <div
+          class="flex flex-col py-2 px-10 text-center lg:w-5/12 lg:text-left"
         >
+          <h1 class="text-very-dark-desaturated-violet py-2 text-3xl font-bold">
+            Simple booking
+          </h1>
+          <p class="text-dark-grayish-violet text-sm">
+            Stay in touch with our dog walkers through the chat interface. This
+            makes it easy to discuss arrangements and make bookings. Once the
+            walk has been completed you can rate your walker and book again all
+            through the chat.
+          </p>
+          <div class="relative">
+            <div class="absolute mt-12 flex w-full flex-col text-center">
+              <div>
+                <a
+                  class="text-optimum-blue hover:text-optimum-darkblue"
+                  href="https://www.frontendmentor.io/challenges/chat-app-css-illustration-O5auMkFqY"
+                  target="_blank"
+                  >Frontend Mentor challenge</a
+                >
+              </div>
+              <div>
+                for
+                <a
+                  class="text-optimum-blue hover:text-optimum-darkblue"
+                  href="https://github.com/optimumBA"
+                  target="_blank"
+                  >Optimum BH</a
+                >
+                Internship.
+              </div>
+              <div class="mt-3">
+                <a
+                  class="text-optimum-blue hover:text-optimum-darkblue"
+                  href="https://github.com/optimumBA/frontend_mentor_challenges/tree/chat_app_css_illustration/challenges/chat_app_css_illustration"
```

Comment:
Put your username instead of `optimumBA`.

Reply:
Noted. I'll make the changes. Thanks.